src/sonic-config-engine/minigraph.py

```diff
@@ -749,6 +749,17 @@ def filter_acl_table_bindings(acls, neighbors, port_channels, sub_role):
 
     return filter_acls
 
+def enable_internal_bgp_session(bgp_sessions):
+    '''
+    In Multi-NPU session the internal sessions will always be up.
+    So adding the admin-status 'up' configuration to bgp sessions
+    '''
+
+    for peer_ip in bgp_sessions.keys():
+        session_name = bgp_sessions[peer_ip]['name']
+        if session_name.startswith(INTERNAL_BGP_SESSION_NAME_PREFIX):
```

Comment:
instead of hard coding the session name prefix, can we check if the bgp session peer role is between FrontEnd and BackEnd? or FrontEnd and FrondEnd to decide if it is internal bgp session or not.

Reply:
Fixed in the latest commit